README.md

```diff
@@ -1,12 +1,12 @@
-# Semantic Environment Mapping, Processing and Reasoning (SEMPR)
```

Comment:
So why is Environment removed from the name?
Last time I've used "Semantic Mapping, Processing and Reasoning" by mistake, I was told the name is "Semantic Environment Mapping, Processing and Reasoning"

Reply:
This is indeed an error: The remove_gdal branch is based on master one commit **before** I fixed the title. :wink: